neo/Core/Helper.cs

```diff
@@ -81,5 +82,21 @@ internal static bool VerifyScripts(this IVerifiable verifiable)
             }
             return true;
         }
+
+        internal static bool VerifyReceivingScripts(this Transaction transaction)
+        {
+            HashSet<ContractState> receiving_contracts = new HashSet<ContractState>(transaction.Outputs.Select(o => Blockchain.Default.GetContract(o.ScriptHash)).Where(c => c != null));
+            foreach (ContractState receiving_contract in receiving_contracts)
+            {
+                using (StateReader service = new StateReader())
+                {
+                    ApplicationEngine engine = new ApplicationEngine(TriggerType.VerificationR, transaction, Blockchain.Default, service, Fixed8.Zero);
+                    engine.LoadScript(receiving_contract.Script, false);
```

Comment:
you just load the contract script, but may be it need some parameter.

Reply:
why are you reviewing the unfinished PR haha